src/tests/ftest/control/daos_admin_privileged.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2020 Intel Corporation.
```

Comment:
2020 shall be enough.

Reply:
I can change this to just 2020 in one of my other PRs if ok, so that I can spare having to run another CI cycle.

Reply:
Sure